danuluma/dannstore
# #161215691 setup app structure

#### What does this PR do?
It set up the app structure

#### Description of Task to be completed?
Set up the app structure
Create a test endpoint 

#### How should this be manually tested?
* This project requires python3 installed. Please install it from https://www.python.org/downloads/
if you don't have it already.
* download https://github.com/danuluma/dannstore/archive/ch-app-structure-161215691.zip
* extract using your favourite archive manager to any location on your computer
* Alternatively, clone the repo by pasting ```git clone https://github.com/danuluma/dannstore.git``` into your terminal or cmd (you need to have git installed if on windows)
* in a terminal, navigate to the extracted\cloned folder location
* run the app by typing ```python3 run.py``` into the terminal
* use postman to access http://localhost:5000/dann/api/v1/home 
* you should get a response in json format like this : 
```
{
    "message": "Hello, there ;-)"
}
```

#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#161215691

#### Screenshots (if appropriate)
#### Questions:
